FacebookSDKStrings.bundle/Resources/hr.lproj/FacebookSDK.strings

```diff
@@ -26,7 +26,7 @@
 "LikeButton.Like" = "Sviđa mi se";
 
 /* The label for the FBSDKLikeButton when the object is currently liked. */
-"LikeButton.Liked" = "Označeno sa "sviđa mi se"";
+"LikeButton.Liked" = 'Označeno sa "sviđa mi se"';
```

Comment:
I don't think this is a valid string for the `strings` format. It doesn't allow for the use of single quotes.